observations/README.md

```diff
@@ -0,0 +1,1279 @@
+| valueQuantity.extension\[0].value                | fixed value = `true` <sup>1</sup>                                                                                                                        |
+| valueString                                      | `ObservationStatement / value` where valueQuantity is not populated                                                                                      |
+| effectiveDateTime                                | `ObservationStatement / effectiveTime` or else `ObservationStatement / availibiltyTime [@value]`                                                         |
+| issued                                           | `ehrCompostion / author / time [@value]` <sup>2</sup> or else `EhrExtract / availibilityTime [@value]`                                                   |
+| performer\[0]                                    | Practitioner referenced in `ObservationStatement / paticipant` <sup>3</sup>                                                                              |
+| interpretation.coding\[0].code                   | `ObservationStatement / interpretationCode [@code]`                                                                                                      |
+| interpretation.coding\[0].display                | `ObservationStatement / interpretationCode [@code]`                                                                                                      |
+| interpretation.coding\[0].system                 | fixed value = `"http://hl7.org/fhir/v2/0078"`                                                                                                            |
+| interpretation.text                              | `ObservationStatement / interpretationCode / originalText` or else `ObservationStatement / interpretationCode [@displayName]`                            |                                         
+| comment                                          | Concatenated from `ObservationStatement / subject / personalRelationship / code` and `ObservationStatement / pertinentInformation / pertinentAnnotation` |
+| referenceRange\[index].text                      | `ObservationStatement / referenceRange[index] / referenceInterpretationRange / text`                                                                     |
+| referenceRange\[index].high                      | `ObservationStatement / referenceRange\[index] / referenceInterpretationRange / value / high`                                                            |
+| referenceRange\[index].low                       | `ObservationStatement / referenceRange\[index] / referenceInterpretationRange / value / low`                                                             |
+
+<details>
+<summary>Example JSON</summary>
```

Comment:
Minor: I found the `valueQuantity` confusing, so having that part of the example might be useful to others.

Reply:
Agreed, I've changed the example to one with `valueQuantity`, `interpretation`, `comment` and `referenceRange`